frontend/front/src/pages/Admin/home/HomeTable.js

```diff
@@ -33,10 +36,19 @@ const HomeTable = () => {
 
   const handleHomeLink = (home) => goToBreadcrumb("home", home);
 
+  const handleUserLink = (home) => {
+    navigate("/admin/survey/visit/" + home.survey_visit_ids[0]);
```

Comment:
Use string interpolation here, e.g.
```
`/admin/survey/visit/${home.survey_visit_ids[0]}`
```

Reply:
Also, not sure how we should be handling survey visits > 1, but I think your approach here is correct.

Reply:
I have trouble hearing in the meetings and wish there was a source of truth on this. Are they still allowing more then one survey?

Maybe if there more then one have a collapsible section that will drown down? We may not be able to do that with the DataGrid mui component, but something on those lines.

Reply:
Changed to string interpolation.